packages/vbet/vbet/vbet.py

```diff
@@ -232,35 +246,36 @@ def vbet(huc, flowlines_orig, flowareas_orig, orig_slope, json_transforms, orig_
                 rasterio.open(chan_dist_transform_raster, 'w', **out_meta) as chan_dist_ev_out, \
                 rasterio.open(fa_dist_transform_raster, 'w', **out_meta) as fa_dist_ev_out:
 
-            progbar = ProgressBar(len(list(slp_src.block_windows(1))), 50, "Calculating evidence layer")
+            progbar = ProgressBar(len(list(open_rasters['Slope'].block_windows(1))), 50, "Calculating evidence layer")
             counter = 0
             # Again, these rasters should be orthogonal so their windows should also line up
-            for _ji, window in slp_src.block_windows(1):
+            for _ji, window in open_rasters['Slope'].block_windows(1):
                 progbar.update(counter)
                 counter += 1
-                slope_data = slp_src.read(1, window=window, masked=True)
-                hand_data = hand_src.read(1, window=window, masked=True)
-                cdist_data = cdist_src.read(1, window=window, masked=True)
-                fadist_data = fadist_src.read(1, window=window, masked=True)
-                da_slope_zones_data = da_slope_zone_src.read(1, window=window, masked=True)
-                da_hand_zones_data = da_hand_zone_src.read(1, window=window, masked=True)
-
-                slope_transform_zones = [np.ma.MaskedArray(transforms["Slope"](slope_data.data), mask=slope_data.mask),
-                                         np.ma.MaskedArray(transforms["Slope MID"](slope_data.data), mask=slope_data.mask),
-                                         np.ma.MaskedArray(transforms["Slope LARGE"](slope_data.data), mask=slope_data.mask)]
-
-                hand_transform_zones = [np.ma.MaskedArray(transforms["HAND"](hand_data.data), mask=hand_data.mask),
-                                        np.ma.MaskedArray(transforms["HAND MID"](hand_data.data), mask=hand_data.mask),
-                                        np.ma.MaskedArray(transforms["HAND LARGE"](hand_data.data), mask=hand_data.mask)]
+                slope_data = open_rasters['Slope'].read(1, window=window, masked=True)
+                hand_data = open_rasters['HAND'].read(1, window=window, masked=True)
+                cdist_data = open_rasters['Channel'].read(1, window=window, masked=True)
+                fadist_data = open_rasters['Flow Areas'].read(1, window=window, masked=True)
+                da_slope_zones_data = open_rasters['Slope_Zone'].read(1, window=window, masked=True)
+                da_hand_zones_data = open_rasters['HAND_Zone'].read(1, window=window, masked=True)
```

Comment:
Maybe a minor point but if your file handles are going to be in a dictionary you could always store the read data the same way no?